content/strategy-goals/strategy/platform/search_platform.md

```diff
@@ -0,0 +1,52 @@
+# Search Platform strategy
```

Comment:
This page seems very outdated. Apart from the link at the top we should probably start from scratch?

Reply:
Alright! Sounds good

Reply:
Done!